### PR TITLE
Besu event handling using web3

### PR DESCRIPTION
**Pull request by rranjan3**
- Downgrade Besu version to 1.3.9. There appears to be some defect in Besu 1.4 onwards 
   around eth_getFilterChanges - hyperledger/besu#473. Reverting to dot release 1.3.9 does
   not show the issue.
- Remove JRPC event handling
- Minor bug fix in event formatting
- Remove redundant library eth-abi from Dockerfile

Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>

### CODE REVIEW

File: docs/dev-environments/besu/docker-compose.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   # node1 is first of the two-node ethereum private network based on Hyperledger Besu
   #
   node1.avalon.org:
-    image: hyperledger/besu:1.4
+    image: hyperledger/besu:1.3.9
```
Comment (danintel) [inline docs/dev-environments/besu/docker-compose.yaml]: Why is Besu being downgraded to 1.3.9?  Add that in the commit message or comment here.

Comment (rranjan3) [reply docs/dev-environments/besu/docker-compose.yaml]: That's already there in the commit message @danintel . Provided link to the filed issue as well.

File: sdk/avalon_sdk/ethereum/ethereum_listener.py [hunk 2]
```diff
@@ -36,127 +31,8 @@ def __init__(self, config):
...
-    This method reads "jsonrpc" field to detect the event source and format
-    it to give all events a similar structure
-    """
-    try:
-        jrpc = evt["jsonrpc"]
-    except KeyError as e:
-        return evt
-    return parse_jrpc_evt(evt, wo_submitted_hash, wo_completed_hash)
+        
+        return contract.events[event].createFilter(fromBlock=fromBlock)
```
Comment (manojgop) [inline sdk/avalon_sdk/ethereum/ethereum_listener.py]: Please add a comment about this filter . For example,"setting up filter to get any events from latest block committed in block chain "

Comment (rranjan3) [reply sdk/avalon_sdk/ethereum/ethereum_listener.py]: Done.

**Pull request merged**